engines/cryomni3d/versailles/menus.cpp

```diff
@@ -99,9 +99,9 @@ uint CryOmni3DEngine_Versailles::displayOptions() {
 	showMouse(true);
 
 	uint hoveredBox = -1;
-	uint selectedBox;
+	uint selectedBox = 0;
```

Comment:
I would put -1 on this one as 0 is a valid box.

Reply:
You can't put a -1 here, unless selectedBox is changed to be signed

Reply:
@bluegr, sorry I didn't see you commented on the issue before merging.

I did it everywhere in the code without any warning from the compiler. It sets the value to UINT_MAX.
It seems to be perfectly legal in C: https://stackoverflow.com/a/50632 so I would say it's legal in C++ too.

Reply:
Yes, setting an unsigned int to a negative value is legal, but you might get yet another warning. ;-)